### DISCUSSION TITLE
Doc on AttestationPolicies Filters (conditions)

### DISCUSSION THREAD

**Discussion by ChakshuGautam** [Sunbird-RC/community#216] [Issues] [answered]
I was only able to find just one example of [this](https://github.com/Sunbird-RC/sunbird-rc-core/blob/4f6fb18d999b00b1d23a4fc880bec6af9d026df4/tools/cli/src/templates/config/schemas/student.json#L38). Pointing to relevant code would also do.

**Comment by pramodkvarma** [comment]
@parthlawate @tejash-jl we should definitely improve docs. 

@chakshugautam I hope you can help contribute given you are looking at adoption and have an adopter perspective.

**Reply by radhay-samagra** [reply]
@pramodkvarma I would like to add another similar issue on top of this, for attestation policies, I have seen **attestorEntity** mentioned in the docs [here](https://github.com/Sunbird-RC/community/blob/main/developer-documentation/creating-your-own-schemas.md?plain=1#L82) to set which entity can attest the claim, but I have nowhere found the implementation details on **attestorPlugin** attribute which I found out [here](https://github.com/Sunbird-RC/demo-education-registry/blob/main/schemas/Teacher.json#L90). Similar issues with property **attestationProperties**.  Can you point me out to a updated doc mentioning all of these attributes?

**Comment by ChakshuGautam** [comment, marked as answer]
I found more docs [here](https://docs.sunbirdrc.dev/use/schema-configuration#attestation-policy). Closing it for now. If these don't suffice will document. 

@tejash-jl would be great if you can share the relevant code here as well. Thanks.

**Comment by tejash-jl** [comment]
@ChakshuGautam @radhay-samagra I'll update the documentation on Attestation policies and share it here once it is done.

**Reply by pramodkvarma** [reply]
Super! Thanks @tejash-jl
